WordPress/Classes/Services/Reader Post/ReaderPostService.h

```diff
@@ -199,3 +202,5 @@ extern NSString * const ReaderPostServiceToggleSiteFollowingState;
 - (BOOL)canLoadMorePostsForTopic:(ReaderAbstractTopic * _Nonnull)readerTopic remotePosts:(NSArray * _Nonnull)remotePosts inContext: (NSManagedObjectContext * _Nonnull)context;
```

Comment:
This is the only API with nullability annotation.

I'm happy to work on annotating the rest of the header, but I think for the time being it's more effective to silence the warning. WDYT?